guides/common/assembly_configuring-capsule-custom-server-certificate.adoc

```diff
@@ -13,8 +13,14 @@ To configure your {SmartProxyServer} with a custom certificate, complete the fol
 . xref:deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[]
 . xref:deploying-a-custom-ssl-certificate-to-hosts_{smart-proxy-context}[]
 
+:ProductName: {SmartProxyServer}
```

Comment:
This is defined at the parent level (in `master.adoc`). So let's remove `:ProductName:` and `:!ProductName:` in this assembly. It should fix the build of the _Installing Proxy_ guide.